src/main/res/layout/activity_server_files.xml

```diff
@@ -19,7 +19,12 @@
   ~ along with Amahi. If not, see <http ://www.gnu.org/licenses/>.
   -->
 
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout
```

Comment:
Keep this as `LinearLayout` with orientation vertical, this way you will not have to add margins to all the ViewAnimators present in the layouts of all the fragments.